lib/spree/localized_number.rb

```diff
@@ -37,6 +29,31 @@ def localize_number(*attributes)
       end
     end
 
+    def override_setter_for(attribute)
+      setter = "#{attribute}="
+      old_setter = instance_method(setter) if table_exists? && !column_names.include?(attribute.to_s)
```

Comment:
I fail to see what's this case. An AR attribute not baked by a database table column?

Reply:
I think it is for preferences? Spree's calculators all share the same table, but they have preferences which are stored in a separate table, using the old setter means that we still set the preference properly. @ltrls should be able to confirm...